src/main.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+class Todo  {
+    constructor() {
+        this.todoPoints = [];
+    }
+
+    addItem(title, priority) {
+        let todoPoint = {};
+        todoPoint.title = title;
+        todoPoint.priority = priority;
+        todoPoint.id = +Math.random().toString().substr(2, 4);
+        this.todoPoints.push(todoPoint);
+        return todoPoint.id
+    }
+
+    getItem(id) {
+        let item = this.todoPoints.filter(item => item.id === id);
+        if (item.length <= 0) {
+            return null;
+        } else {
+            return item;
+        }
+    }
+
+    removeItem(id) {
+        let item = this.todoPoints.filter(item => item.id === id);
+        this.todoPoints = this.todoPoints.filter(item => item.id !== id);
+        return item.length > 0;
```

Comment:
`... it should return true if the item with the given id was successfully removed or false if there was no such item;`